node/src/index.ts

```diff
@@ -51,6 +53,24 @@ export interface ConnectionOptions {
   hostOverride?: string
 }
 
+function appendAwsCreds (
+  callArgs: any[],
+  awsCredentials?: AwsCredentials,
+  awsRegion?: string
+): void {
+  if (awsCredentials !== undefined) {
+    callArgs.push(awsCredentials.accessKeyId)
+    callArgs.push(awsCredentials.secretKey)
+    callArgs.push(awsCredentials.sessionToken)
```

Comment:
Was the removal of the `if (awsCredentials.sessionToken !== undefined)` check intentional?

Reply:
yes, not pushing and pushing `undefined` gets us the same behavior rust side. We need to start pushing `undefined` because we need to use the position to identify the argument. We probably need to do some refactoring with arg passing across FFI as the current pattern is pretty messy.